src/Symfony/Component/Messenger/composer.json

```diff
@@ -25,7 +25,8 @@
         "symfony/property-access": "~3.4|~4.0",
         "symfony/var-dumper": "~3.4|~4.0",
         "symfony/property-access": "~3.4|~4.0",
-        "symfony/process": "~4.0"
+        "symfony/process": "~4.0",
+        "symfony/validator": "~4.0"
```

Comment:
would 3.4 work (same for process?)
can you fix the alpha order also? (this is a new component so it's still time to fix this)